test/js/specs/happy.js

```diff
@@ -208,4 +208,206 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
+          crossDevice.verifyCrossDeviceNumberInput()
+          crossDevice.verifyCrossDeviceSendLinkBtn(crossDeviceSyncCopy)
+          crossDevice.verifyCrossDeviceCopyLinkInstead(crossDeviceSyncCopy)
+          crossDevice.verifyCrossDeviceCopyToClipboardBtn(crossDeviceSyncCopy)
+          crossDevice.verifyCrossDeviceCopyLinkTextContainer()
+          crossDevice.verifyCrossDeviceDivider()
+        })
+
+        it('should change the state of the copy to clipboard button after clicking', async () => {
+          driver.get(localhostUrl + `?language=${lang}`)
+          const crossDeviceSyncCopy = documentSelection.copy(lang)
+          goToCrossDeviceScreen()
+          crossDevice.crossDeviceCopyToClipboardBtn.click()
+          crossDevice.verifyCrossDeviceCopyToClipboardBtnChangedState(crossDeviceSyncCopy)
+        })
+
+        it('should display error when number is not provided', async () => {
+          driver.get(localhostUrl + `?language=${lang}`)
+          const crossDeviceSyncCopy = documentSelection.copy(lang)
+          goToCrossDeviceScreen()
+          crossDevice.typeMobileNumebr('123456789')
+          crossDevice.crossDeviceSendLinkBtn.click()
+          crossDevice.verifyCrossDeviceCheckNumberCorrectError(crossDeviceSyncCopy)
+        })
+
+        it('should display error when number is wrong', async () => {
+          driver.get(localhostUrl + `?language=${lang}`)
+          const crossDeviceSyncCopy = documentSelection.copy(lang)
+          goToCrossDeviceScreen()
+          crossDevice.typeMobileNumebr('123456789')
+          crossDevice.crossDeviceSendLinkBtn.click()
+          crossDevice.verifyCrossDeviceCheckNumberCorrectError(crossDeviceSyncCopy)
+        })
+
+        it('should send sms and navigate to check your mobile screen ', async () => {
+          const crossDeviceCheckYourMobileCopy = crossDeviceCheckYourMobile.copy(lang)
+          driver.get(localhostUrl + `?language=${lang}`)
+          goToCrossDeviceScreen()
+          crossDevice.typeMobileNumebr(testDeviceMobileNumber)
+          crossDevice.crossDeviceSendLinkBtn.click()
+          waitForAlertToAppearAndSendSms()
+          crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileTitle(crossDeviceCheckYourMobileCopy)
+        })
+
+        describe('cross device check your mobile screen', async () => {
+
+          it('should verify UI elements of the cross device check your mobile screen', async () => {
+            driver.get(localhostUrl + `?language=${lang}`)
+            const crossDeviceCheckYourMobileCopy = crossDeviceCheckYourMobile.copy(lang)
+            goToCrossDeviceScreen()
+            crossDevice.typeMobileNumebr('07495 023357')
+            crossDevice.crossDeviceSendLinkBtn.click()
+            waitForAlertToAppearAndSendSms()
+            crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileTitle(crossDeviceCheckYourMobileCopy)
+            if (lang === 'en') {
```

Comment:
Why do you need this conditional statement here? And why do you need to hard code the copy when you can use `crossDeviceCheckYourMobileCopy`?

Reply:
I've hardcoded the copy as the copy in en.json and es.json is `"submessage": "We’ve sent a secure link to %{number}"`

The method that verifies the elementCopy is taking the text from the json and compares with the string so 
`We’ve sent a secure link to %{number}` won't match `We’ve sent a secure link to 07495 023357` and test will fail.